### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by kedrap**

### CODE REVIEW

File: bundles/SyliusResourceBundle/configuration.rst [hunk 1]
```diff
@@ -21,7 +21,7 @@ In your ``app/config.yml`` (or in an imported configuration file), you need to d
                 object_manager: default
                 templates: App:User
                 classes:
-                    model: MyApp\Entity\EntityName
+                    model: MyApp\Entity\EntityKey
```
Comment (aRn0D) [inline bundles/SyliusResourceBundle/configuration.rst]: This should be reverted

Comment (aRn0D) [inline bundles/SyliusResourceBundle/configuration.rst]: You should remane OtherEntityKey

**Comment by aRn0D** [comment]
@kedrap Thanks for your contribution, could you squash your commits plz?

**Comment by kedrap** [comment]
Sure, I will squash it.

But one question: should I revert line 'model: MyApp\Entity\EntityKey' to 'model: MyApp\Entity\EntityName' as originally, or rename it to 'model: MyApp\Entity\OtherEntityKey'?

My intention was to make it consistent. When we have resource 'my_app.entity_key:' we should have 'model: MyApp\Entity\EntityKey'. When we have 'my_app.other_entity_key:' we should have 'model: MyApp\Entity\OtherEntityKey'. Don't you think so?

**Comment by aRn0D** [comment]
You should suffix the class of the model  by Name and the resource identifier by Key because you are not obliged to use the class of the model as identidier.

**Comment by kedrap** [comment]
Done

**Comment by aRn0D** [comment]
Thank you Piotr

**Pull request merged**